zipkin-lens/src/components/App/App.jsx

```diff
@@ -33,41 +32,41 @@ import { i18n } from '../../util/locale';
 
 import { BASE_PATH } from '../../constants/api';
 
+import AlertSnackbar from './AlertSnackbar';
+
 const App = () => {
   useDocumentTitle('Zipkin');
   return (
     <Suspense fallback={<CircularProgress />}>
       <UiConfig>
         <MuiPickersUtilsProvider utils={MomentUtils}>
           <ThemeProvider theme={theme}>
-            {/* Snackbar is used to provide popup alerts to the user */}
-            <SnackbarProvider>
-              <UiConfigConsumer>
-                {(config) => (
-                  <Provider store={configureStore(config)}>
-                    <I18nProvider i18n={i18n}>
-                      <BrowserRouter basename={BASE_PATH}>
-                        <Layout>
-                          <Route exact path="/" component={DiscoverPage} />
-                          {config.dependency.enabled && (
-                            <Route
-                              exact
-                              path="/dependency"
-                              component={DependenciesPage}
-                            />
-                          )}
+            <UiConfigConsumer>
+              {(config) => (
+                <Provider store={configureStore(config)}>
+                  <AlertSnackbar />
+                  <I18nProvider i18n={i18n}>
+                    <BrowserRouter basename={BASE_PATH}>
+                      <Layout>
+                        <Route exact path="/" component={DiscoverPage} />
+                        {config.dependency.enabled && (
                           <Route
                             exact
-                            path={['/traces/:traceId', '/traceViewer']}
-                            component={TracePage}
+                            path="/dependency"
```

Comment:
alphabetically ordering the routes, eh? :D

Reply:
This is GitHub rendering an indentation change poorly, but no change to the routes :)